EXAMPLES.md

```diff
@@ -217,6 +214,47 @@ cart.LineItems.Get(firstProduct, selectedOptions);
 cart.LineItems.Delete(firstProduct, selectedOptions);
 ```
 
+### Native web view checkout
+
+After creating an instance of `Cart` and adding items to it, you can use the `CheckoutWithNativeWebView` method to 
```

Comment:
Later on we should maybe mention for iOS and Android. Since potentially someone maybe hitting another platform.